src/graphql/outlier.rs

```diff
@@ -563,12 +563,13 @@ where
             map.iter_backward()?
         };
 
-        let (nodes, has_more) = if let Some(after) = after {
+        let (mut nodes, has_more) = if let Some(after) = after {
             let to = earliest_outlier_key(&after)?;
             iter_to_nodes_with_outlier(iter, &to, cmp::Ordering::is_ge, last, filter, true)
         } else {
             iter_to_nodes_with_outlier(iter, &[], always_true, last, filter, true)
         }?;
+        nodes.reverse();
```

Comment:
It would also be beneficial if we have a test on `outliers` API, that looks very similar to `check_customer_ordering`, so that we are sure that edges in the `outliers` graphql API response is in ascending order after this change.

Reply:
Sounds good!

Reply:
From what I have tested the current version without this change, I've got proper results. So did I with other APIs like `clusters`, `eventList`, and `models`. This means this PR is not necessary any longer.

![스크린샷 2024-02-26 오후 8 00 35](https://github.com/aicers/review-web/assets/55010932/c6de9a8c-66a7-444e-9d37-a18ca08f8df3)
![스크린샷 2024-02-26 오후 8 01 04](https://github.com/aicers/review-web/assets/55010932/549411f8-02f0-42b2-b87e-9e2b92a7c1d6)
![스크린샷 2024-02-26 오후 8 01 19](https://github.com/aicers/review-web/assets/55010932/e7936c0b-3795-4e67-b370-8f3143073e20)
![스크린샷 2024-02-26 오후 8 01 50](https://github.com/aicers/review-web/assets/55010932/d6d99ad8-5b38-47b6-b56d-0bd85e28ebb3)

Reply:
Could you close this?

Reply:
If other APIs rather than `customerList` need to be tested due to the difference how they use the change, I think you should test those as well.